read.js

```diff
@@ -0,0 +1,37 @@
+import { success, failure } from "./libs/response-lib";
+import * as dynamoDbLib from "./libs/dynamodb-lib";
+import User from './model/user';
+import HttpStatus from 'http-status-codes';
+import { UUID_V1 } from './validators/user';
+
+/**
+ * Read API Lambda for User model.
+ */
+export async function main(event, context) {
+  try {
+    if (!('pathParameters' in event)) {
+      return failure(HttpStatus.UNPROCESSABLE_ENTITY, {error : "ID Path Parameter required."});
+    }
+    const id = event.pathParameters.id;
+
+    if (!id) {
+      return failure(HttpStatus.UNPROCESSABLE_ENTITY, {error : "ID Path Parameter required."});
+    } else if (!id.match(UUID_V1)) {
+      throw {message : "Required UUID V1 format."};
+    }
+
+    let user = new User(dynamoDbLib);
```

Comment:
I'm not sure what this pattern of passing `dynamoDbLib` is achieving. Looks like you could just import it inside of User (or any other such classes).

Reply:
This is to mock the functions from dynamoDbLib for testability, its dependency injection.  It is simpler from the unit test side of things to mock a function then to mock an import.